presto-native-execution/CMakeLists.txt

```diff
@@ -198,6 +201,10 @@ if(PRESTO_ENABLE_JWT)
   add_compile_definitions(PRESTO_ENABLE_JWT)
 endif()
 
+if(PRESTO_ENABLE_PROMETHEUS_REPORTER)
```

Comment:
This block can be removed. We don't use `ifdef PRESTO_ENABLE_PROMETHEUS_REPORTER` anymore.